src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java

```diff
@@ -1044,7 +1044,7 @@ else if(model instanceof AcogReticle) {
                   }, 250, 0))
                   
           .build())
-		.withSpawnEntityDamage(7f)
+		.withSpawnEntityDamage(10f)
```

Comment:
please revert this change (I will do extended research on balance of weapons at a later date)